tests/test_files/valid_bundles/results/bundle.2.yaml

```diff
@@ -0,0 +1,206 @@
+data:
```

Comment:
This file seems to be identical to `tests/test_files/valid_bundles/results/bundle.1.yaml`, why does it exist?